### PR TITLE
Add a test to verify `tag_study_user_attrs` for `MLflowCallback` works properly

### PR DESCRIPTION
**Pull request by harupy**
## Motivation

A follow-up for #1918

## Description of the changes

Add a test to verify `tag_study_user_attrs` for `MLflowCallback` works properly.

### CODE REVIEW

File: tests/integration_tests/test_mlflow.py [hunk 1]
```diff
@@ -132,3 +132,32 @@ def test_tag_truncation(tmpdir: py.path.local) -> None:
...
     my_user_attr = first_run_dict["data"]["tags"]["my_user_attr"]
     assert len(my_user_attr) <= 5000
+
+
+def test_tag_study_user_attrs(tmpdir: py.path.local) -> None:
+    tracking_file_name = "file:{}".format(tmpdir)
+    study_name = "my_study"
+    n_trials = 3
+
+    mlflc = MLflowCallback(tracking_uri=tracking_file_name, tag_study_user_attrs=True)
```
Comment (hvy) [inline tests/integration_tests/test_mlflow.py:142]: How about testing that it's not there when the parameter is `False`?

File: tests/integration_tests/test_mlflow.py [hunk 2]
```diff
@@ -132,3 +132,32 @@ def test_tag_truncation(tmpdir: py.path.local) -> None:
...
+    assert len(experiments) == 1
+
+    experiment = experiments[0]
+    assert experiment.name == study_name
+    experiment_id = experiment.experiment_id
+
+    run_infos = mlfl_client.list_run_infos(experiment_id)
+    assert len(run_infos) == n_trials
+
+    first_run_id = run_infos[0].run_id
```
Comment (hvy) [inline tests/integration_tests/test_mlflow.py:158]: A detail but we could also loop to really make sure.

**Review by hvy — COMMENTED** [review]
Thanks a lot for your follow up PRs. I left some huge nits.

File: tests/integration_tests/test_mlflow.py (hunk 1 above)
Comment (harupy) [reply tests/integration_tests/test_mlflow.py]: Addressed in https://github.com/optuna/optuna/pull/1935/commits/a80e9a5d5b3fa298e11126a6e17aa7248d870b03

File: tests/integration_tests/test_mlflow.py (hunk 2 above)
Comment (harupy) [reply tests/integration_tests/test_mlflow.py]: Addressed in https://github.com/optuna/optuna/pull/1935/commits/a80e9a5d5b3fa298e11126a6e17aa7248d870b03

**Review by hvy — APPROVED** [review]
LGTM!

**Review by HideakiImamura — APPROVED** [review]
Thanks for the PR. LGTM!

**Pull request merged**